lib/screens/form_screen.dart

```diff
@@ -38,21 +55,49 @@ class FormScreen extends StatelessWidget {
                   SizedBox(
                     height: SPACING_16,
                   ),
-                  InputField(
-                    onChanged: (text) {},
-                    focusNode: nameFocusNode,
-                    labelText: 'Número da placa antiga',
-                    isValid: true,
+                  Stack(
+                    alignment: Alignment.centerRight,
+                    children: <Widget>[
+                      InputField(
+                        onChanged: (text) {},
+                        focusNode: oldNumberFocusNode,
+                        labelText: 'Número da placa antiga',
+                        isValid: true,
+                        controller: oldNumberController,
+                      ),
+                      IconButton(
+                        icon: Image.asset(
+                          'lib/assets/icons/barcode-scanner.png',
+                        ),
+                        onPressed: () async {
+                          oldNumberController.text = await handleScan();
+                        },
+                      ),
+                    ],
                   ),
                   SizedBox(
                     height: SPACING_16,
                   ),
-                  InputField(
-                    validator: (value) => requiredValidator(value),
-                    onChanged: (text) {},
-                    focusNode: cnpjFocusNode,
-                    labelText: 'Nova placa',
-                    isValid: true,
+                  Stack(
+                    alignment: Alignment.centerRight,
+                    children: <Widget>[
+                      InputField(
+                        validator: (value) => requiredValidator(value),
+                        onChanged: (text) {},
+                        focusNode: newNumberFocusNode,
+                        labelText: 'Nova placa',
+                        isValid: true,
+                        controller: newNumberController,
+                      ),
+                      IconButton(
+                        icon: Image.asset(
+                          'lib/assets/icons/barcode-scanner.png',
+                        ),
+                        onPressed: () async {
+                          newNumberController.text = await handleScan();
+                        },
```

Comment:
Se for só uma linha, acho melhor deixar assim 😁
```suggestion
                        onPressed: () async => newNumberController.text = await handleScan(),
```